test/integration/docker-selenium/wdio.conf.js

```diff
@@ -1,3 +1,7 @@
+require('@babel/register');
+
+const DockerLauncher = require('../../../src/launcher');
```

Comment:
change path to use transpiled launcher from `../../../lib/launcher`

Reply:
Done. In this case I was a little confused. I wanted to run everything through `babel-node` like `babel-node ./node_modules/.bin/wdio ./test/integration/docker-selenium/wdio.conf.js` so I don't worry about transpilation. But then I realised, since you export the files already transpiled in `lib`, the integration tests have to use those sources.